src/patternfly/components/DataList/data-list-expandable-content-body.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-data-list__expandable-content-body{{#if data-list-expandable-content-body--modifier}} {{data-list-expandable-content-body--modifier}}{{/if}}"
```

Comment:
Looks like you need to update the new datalist demos, too. Comment below in the sidebar change about doing this in one place instead of each demo/example.